wd_demo.au3

```diff
@@ -162,6 +170,21 @@ Func RunDemo($idDebugging, $idBrowsers, $idUpdate, $idHeadless)
 	; Check and set desired headless mode
 	Local $bHeadless = _RunDemo_Headless($idHeadless)
 
+	Local $sOutput = _RunDemo_Output($idOutput)
+
+	Switch $sOutput
+		Case 'ConsoleWrite'
+			_WD_Option('console', ConsoleWrite)
+		Case 'WD_Console.log'
+			_WD_Option('console', @ScriptDir & '\WD_Console.log')
+		Case '_DebugOut'
+			_DebugSetup('wd_demo - console log output')
+			_WD_Option('console', _DebugOut)
+		Case 'Null'
+			_WD_Option('console', Null)
+	EndSwitch
+
+
```

Comment:
A few minor issues observed here --

- This section of code needs to be moved towards the beginning of the function (before other _WD* calls). Otherwise, you get output to both the Scite console and the selected output target.
- _WD_Capabilities doesn't honor the new output target
- `_RunDemo_Output` is essentially a single line of code. If you want to keep the function, then it would make sense to me to move the Switch..EndSwitch into it.

Reply:
Please check current commits.
And https://github.com/Danp2/au3WebDriver/pull/210